_studio/shared/umc/core/umc/include/umc_media_data.h

```diff
@@ -73,7 +73,7 @@ class MediaData
 
     // Get an address of the beginning of valid data.
     // This pointer could not be equal to the beginning of the buffer.
-    virtual void* GetDataPointer(void)   { return m_pDataPointer; }
+    virtual uint8_t* GetDataPointer(void)   { return m_pDataPointer; }
```

Comment:
I would recommend to use cast in place. Such change is harmful, in theory it could silently break inheritance (maybe already have done it)